src/patternfly/assets/images/table--icon-github.hbs

```diff
@@ -0,0 +1,4 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" fill="currentColor" width="30px">
```

Comment:
just a nit - these shouldn't need `table--` in the filename since we can use them globally.